### PR TITLE
get apiserver & etcd certificates and their http health checks

### PR DESCRIPTION
**Pull request by adamancini**
bringing in some of the relevant collectors from https://github.com/replicatedhq/troubleshoot-specs/blob/main/host/default.yaml

### CODE REVIEW

**Review by diamonwiggins — APPROVED** [review]

File: pkg/goods/support/host-support-bundle.yaml [hunk 1]
```diff
@@ -5,116 +5,189 @@ metadata:
...
+              message: No IPv4 interfaces detected
+          - pass:
+              when: 'count >= 1'
+              message: IPv4 interface detected
+    - memory:
+        checkName: Amount of Memory
+        outcomes:
+          - warn:
+              when: '< 2G'
+              message: At least 2G of memory is recommended
```
Comment (chris-sanders) [inline pkg/goods/support/host-support-bundle.yaml:130]: Moving forward with EC, we're going to try not having warnings. Either something is important enough to Fail or it isn't. Choose what you think is a fair solution here.

**Review by chris-sanders — DISMISSED** [review]
There's a lot of changes here that I don't think are relevant. I'm guessing your editor ran this through a linter. Can you not lint/change things _or_ submit the lint as it's own with this change after that for easier review.

I tend to prefer just not running a lint, having different people submit different lint configs isn't very helpful. If we really want linting we should just add an enforced lint rule to the tests so it stays consistent.

File: pkg/goods/support/host-support-bundle.yaml (hunk 1 above)
Comment (adamancini) [reply pkg/goods/support/host-support-bundle.yaml:130]: :thumbsup:

**Review by banjoh — APPROVED** [review]

**Comment by adamancini** [comment]
@chris-sanders fixed the linter changes, and moved some warns to fails

**Review by banjoh — APPROVED** [review]

**Pull request merged**